src/Lykke.MatchingEngine.Connector/Services/TcpMatchingEngineClient.cs

```diff
@@ -33,16 +33,20 @@ private long GetNextRequestId()
                 return _currentNumber++;
         }
 
-        public TcpMatchingEngineClient(IPEndPoint ipEndPoint, ISocketLog socketLog = null)
+        public TcpMatchingEngineClient(IPEndPoint ipEndPoint, ISocketLog socketLog = null, bool ignoreErrors = false)
         {
             _clientTcpSocket = new ClientTcpSocket<MatchingEngineSerializer, TcpOrderSocketService>(
                 socketLog,
                 ipEndPoint,
                 3000,
```

Comment:
A magic number

Reply:
Ok, I'll fix it. But it would be default value as well.